### DISCUSSION TITLE
MeshCentral community in telegram

### DISCUSSION THREAD

**Discussion by lixxdee** [Ylianst/MeshCentral#6277] [Show and tell]
To popularize meshcentral, I created a public group [https://t.me/meshcentral](https://t.me/meshcentral) in telegram. If possible, we will help each other in solving problems or problems that have arisen. I hope Ylianst is not against such a community? :)

**Comment by Ylianst** [comment]
I am not against any community :) I am not a telegram user, but may give it a shot.

**Comment by Ylianst** [comment]
Wow, pretty big group already. Just joined it.

![image](https://user-images.githubusercontent.com/1319013/181823296-bd0bc6e0-fbc2-4a54-88b9-a5393da4bab7.png)

**Comment by si458** [comment]
#4390 added to issue templates to help users
